index.html

```diff
@@ -0,0 +1,180 @@
+<!DOCTYPE html>
+
+<html style="font-size: 10px;font-family:Roboto, Arial, sans-serif" lang="en-GB" dark system-icons typography typography-spacing darker-dark-theme darker-dark-theme-deprecate>
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="import" href="https://googleapis.com/icon?family=Material+Icons" rel="stylesheet"/>
+    <link rel="stylesheet" href="style.css">
+    <title>Your-Tube-MV</title>
+</head>
+<body>
+    <div class="thumbnail">
+                <img src="https://img.youtube.com/vi/zUwB_imVjmg/maxresdefault.jpg" alt=""/>
+            </div>
+    <header class="header">
+        <div class="logo left">
+            <i id="menu" class="material-icons">menu</i>
+            <img src="https://www.freecodecamp.org/news/content/images/2022/01/yt-logo.png" alt="youtube logo">
+        </div>
+        <div class="search center">
+            <form action="">
+                <input type="text" placeholder="Search"/>
+                <button><i class="material-icons">search</i></button>
+            </form>
+            <i class="material-icons mic">mic</i>
+        </div>
+        <div class="icons-right">
+            <i class="material-icons">videocam</i>
+            <i class="material-icons">apps</i>
+            <i class="material-icons">notifications</i>
```

Comment:
The logo files were not added to the VSCode browser which is why they do not display. It is suggested that you must kindly copy the files to the browser. So that they are displayed well. This will add good shape to your page.